src/mixins/createParentListMixin.ts

```diff
@@ -82,7 +77,7 @@ const createParentMixin: ParentListMixinFactory = compose<ParentList<Child>, Par
 			}
 		},
 
-		append(this: ParentListMixin<Child>, child: Child | Child[]): Handle {
+		append(this: ParentListMixin<Child>, child: (Child) | (Child)[]): Handle {
```

Comment:
Because, strangely, this wasn't working right... It equated to `(Child|Child)[]`, but let me double check that again... we could do `Child[] | Child`... either way it felt ambiguous and I thought it caused some problems.  The problem only highlighted when I passed something that actually didn't match `Child` and was getting argument was unassignable because it didn't contain `.length`.